packages/fxa-auth-server/fxa-oauth-server/test/key-management-scripts.js

```diff
@@ -0,0 +1,100 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const { assert } = require('chai');
+const fs = require('fs');
+const os = require('os');
+const path = require('path');
+const { execFileSync } = require('child_process');
+const crypto = require('crypto');
+const rimraf = require('rimraf');
+
+describe('the signing-key management scripts', () => {
+  let runScript;
+  let workDir, keyFile, newKeyFile, oldKeyFile;
+
+  beforeEach(() => {
+    const uniqName = crypto.randomBytes(8).toString('hex');
+    workDir = path.join(os.tmpdir(), `fxa-oauth-server-tests-${uniqName}`);
```

Comment:
I couldn't be bothered trying to decide on which tiny utility library to bring in for making tempdirs, this'll do the job.